apps/styling/src/app/text.component.ts

```diff
@@ -1,16 +1,13 @@
 /* eslint-disable @angular-eslint/component-selector */
-import { Component, Input } from '@angular/core';
+import { Component } from '@angular/core';
 
 @Component({
   selector: 'text',
   standalone: true,
   template: `
-    <p style="font-size: {{ font }}px; color: {{ color }}">
+    <p style="font-size: var(--text-font-size); color: var(--text-font-color)">
```

Comment:
My preference is to keep the style inside the `styles` property of the decorator. 

However you should give a default color to your css var if the css var is never defined.

Reply:
Mine too - I was just too lazy to change it :D 

That is a good hint - thanks =)